css/index.less

```diff
@@ -0,0 +1,160 @@
+/* colors used throughout the page */
+@dark-brown: hsl(31, 49%, 10%);
+@light-brown: lighten(@dark-brown, 50%);
+@near-black: hsl(220, 6%, 10%);
+@red: hsl(0, 100%, 50%);
+@lighter-brown: lighten(@light-brown, 15%);
+/* set box-sizing, margin, padding, and color for html and body */
+html, body {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  margin:0;
+  padding:0;
+  color: @dark-brown;
+}
+/* set background image for the body */
+body {
+  background-image: url('../img/shopping-center-iot.jpg');
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
```

Comment:
Ditto. Unless your targeting older browsers (I think this one is good bavk to IE7?)